lib/sqlalchemy/dialects/postgresql/types.py

```diff
@@ -247,6 +250,17 @@ def as_generic(self, allow_nulltype: bool = False) -> sqltypes.Interval:
     def python_type(self) -> Type[dt.timedelta]:
         return dt.timedelta
 
+    @no_type_check
```

Comment:
**Federico Caselli** (CaselIT) wrote:

here too

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4873

Reply:
**mike bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4873